### PR TITLE
📖 Documents HTML Attribute, implemented in #13712

### PR DESCRIPTION
**Pull request by jonkeller**
Documents #13712

### CODE REVIEW

File: spec/amp-var-substitutions.md [hunk 1]
```diff
@@ -622,6 +623,35 @@ Provides the horizontal scroll boundary that triggered a scroll event. This vari
 * **platform variable**: N/A
 * **amp-analytics variable**: `${horizontalScrollBoundary}`
 
+#### HTML Attributes
+
+Provides values of attributes of HTML elements inside of an amp-ad tag which match a given CSS selector. 
+This is only available within the same amp-ad tag; it will not, for instance, allow a publisher to obtain information
```
Comment (bpaduch) [inline spec/amp-var-substitutions.md]: I'm stumbling on the wording "same amp-ad tag".  Same as what? Can you clarify?
I'm wondering if you want to say "this is confined to the contents of the amp-ad tag;"

File: spec/amp-var-substitutions.md [hunk 2]
```diff
@@ -622,6 +623,35 @@ Provides the horizontal scroll boundary that triggered a scroll event. This vari
...
+Caveats:
+* If the CSS selector matches 20 or more elements, an empty array will be returned. This is because traversing a 
+large list of elements is inefficient. 
+* Attributes of at most 10 elements will be returned.
+* If an element matches the CSS selector but has none of the requested attributes, that element will not be 
+represented in the returned array.
+* The returned values will be in the form of a URL encoded JSON array of objects wherein the object keys are the 
+requested attribute names and the object values are the elements' values for those attributes.
+* The CSS selector may contain only letters (upper- and/or lower-case), numbers, hyphens, underscores, and periods. 
+Issue #14252 has been created to address potential future demand for more complex CSS selectors.
```
Comment (bpaduch) [inline spec/amp-var-substitutions.md]: Link to issue

File: spec/amp-var-substitutions.md (hunk 1 above)
Comment (jonkeller) [reply spec/amp-var-substitutions.md]: Thanks - I agree that's awkward. What I want to say is:
If an amp-ad tag contains an amp-analytics tag, you can use this within the amp-analytics tag to see the content of the HTML that gets loaded by that particular amp-ad tag. This applies even if the amp-analytics tag uses iframe transport.
But, an amp-analytics tag inside of one amp-ad tag can't use this to see the HTML that gets loaded by a different amp-ad tag. Also, an amp-analytics tag that's not within any amp-ad tag can't use this at all.

File: spec/amp-var-substitutions.md (hunk 2 above)
Comment (jonkeller) [reply spec/amp-var-substitutions.md]: Done

File: spec/amp-var-substitutions.md (hunk 1 above)
Comment (jonkeller) [reply spec/amp-var-substitutions.md]: Ok, I took another shot at it :)

**Pull request merged**